src/test/java/nextstep/ladder/LadderTest.java

```diff
@@ -1,20 +1,64 @@
 package nextstep.ladder;
 
 import nextstep.ladder.domain.Ladder;
+import nextstep.ladder.domain.Line;
+import nextstep.ladder.domain.Lines;
+import nextstep.ladder.domain.Point;
 import org.junit.jupiter.api.Test;
 
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 class LadderTest {
 
-    private static final int VALID_MIM_COUNT_OF_PERSON = 2;
-
-    private static final int INVALID_HEIGHT_OF_LADDER = -1;
-
     @Test
     void 사다리는_최소_1이상의_높이를_가져야_한다() {
-        assertThatThrownBy(() -> Ladder.of(VALID_MIM_COUNT_OF_PERSON, INVALID_HEIGHT_OF_LADDER))
+        assertThatThrownBy(() -> Ladder.of(List.of("pobi", "honux"), new Lines(List.of()), List.of("꽝", "5000")))
                 .isInstanceOf(IllegalArgumentException.class)
                 .hasMessage("최소 높이는 최소 1이상이어야 합니다.");
     }
+
+    @Test
+    void 사다리의_플레이어_수와_결과_수는_동일해야_한다() {
+        assertThatThrownBy(() -> Ladder.of(
+                        List.of("pobi", "honux"),
+                        new Lines(List.of(
+                                Line.from(List.of(new Point(false), new Point(false))),
+                                Line.from(List.of(new Point(false), new Point(false)))
+                        ))
+                        , List.of("꽝")
+                )
+        )
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessage("플레이어와 결과의 수는 동일해야 합니다.");
+    }
```

Comment:
assert 메서드에 전달하는 람다식은 테스트하고자 하는 코드만 전달하면 어떨까요?
반복되는 코드도 메서드로 분리하고 테스트하고자 하는 대상을 명확히하면 가독성이 올라가서
테스트의 의도를 이해하는데 도움될 것 같아요

```suggestion
    @Test
    void 사다리의_플레이어_수와_결과_수는_동일해야_한다() {
        final List<String> playerNames = List.of("pobi", "honux");
        final List<String> result = List.of("꽝");

        final Line line = lineOf(false, false);
        final Lines lines = new Lines(List.of(line, line));

        assertThatThrownBy(() -> Ladder.of(playerNames, lines, result))
            .isInstanceOf(IllegalArgumentException.class)
            .hasMessage("플레이어와 결과의 수는 동일해야 합니다.");
    }

    private Line lineOf(boolean first, boolean second) {
        return Line.from(List.of(new Point(first), new Point(second)));
    }
```